### DISCUSSION TITLE
Where's the latest API documented?

### DISCUSSION THREAD

**Discussion by Crayder** [Tampermonkey/tampermonkey#1375] [Q&A] [answered]
I can't find info anywhere! `GM.info` doesn't exist, `GM.getResourceUrl` doesn't exist, and I'm not entirely sure what all does exist anymore basically. The changelog for 4.0+ versions mentions things like `GM_xhr`, `GM_addElement`,  and `GM_setStyle`... but where are the docs! What do they do! What else exists that I don't know about?

**Comment by derjanb** [comment, marked as answer]
Are you searching for this: https://www.tampermonkey.net/documentation.php? `GM.info` is `GM_info`, `GM.getResourceUrl` is `GM_getResourceURL` and so on.

**Reply by pqhchan** [reply]
And where can find the description for the GM_webRequest ?

**Reply by derjanb** [reply]
This API is experimental and will likely go away with mv3 next year. 
@7nik created a nice summary here: https://github.com/Tampermonkey/tampermonkey/issues/397#issuecomment-909169728

**Comment by Crayder** [comment]
Yes, but it's missing things (EDIT: like `GM_xhr` and an explanation of how to use `GM_info` perhaps). And GM_Info just wasn't doing anything. It is now so I'm guessing that was either particular to my work machine or the version of the same script on my work machine.

**Comment by Martii** [comment]
@derjanb 

Is `GM_xhr` a slang abbreviation for [`GM_xmlhttpRequest`](https://www.tampermonkey.net/documentation.php#GM_xmlhttpRequest) or is it something else *(perhaps in a beta)*?

Thanks for the look,
OUJS Official Business

**Reply by derjanb** [reply]
> Is `GM_xhr` a slang abbreviation for [`GM_xmlhttpRequest`](https://www.tampermonkey.net/documentation.php#GM_xmlhttpRequest)

Yes, exactly.
